docs/source/developer_guides/metrics.md

```diff
@@ -26,22 +26,24 @@ At the core of this module is a centralized registry, `MetricRegistry`, which st
   - `embedding`: Silhouette Score
   - `integration`: Entropy per Cell, Batch Silhouette
   - `label_prediction`: Accuracy, F1, Precision, Recall, AUROC
-  - `perturbation`: MSE, R², Jaccard Similarity
+  - `perturbation`: Spearman correlation, Accuracy, F1, Precision, Recall
 
 ## Supported Metrics
 
 The following metrics are pre-registered:
 
-| **Metric Type**          | **Task**     | **Description**                                                                                                  |
+| **Metric Type**          | **Task**         | **Description**                                                                                                  |
 |--------------------------|------------------|------------------------------------------------------------------------------------------------------------------|
 | `adjusted_rand_index`    | clustering       | Measures the similarity between two clusterings, adjusted for chance. A higher value indicates better alignment. |
 | `normalized_mutual_info` | clustering       | Quantifies the amount of shared information between two clusterings, normalized to ensure comparability.         |
 | `silhouette_score`       | embedding        | Evaluates how well-separated clusters are in an embedding space. Higher scores indicate better-defined clusters. |
 | `entropy_per_cell`       | integration      | Assesses the mixing of batch labels at the single-cell level. Higher entropy indicates better integration.       |
 | `batch_silhouette`       | integration      | Combines silhouette scoring with batch information to evaluate clustering quality while accounting for batch effects. |
-| `mean_squared_error`     | perturbation     | Calculates the average squared difference between predicted and true values, indicating prediction accuracy.     |
-| `r2_score`               | perturbation     | Measures the proportion of variance in true values explained by predictions. Higher values indicate better predictions. |
-| `jaccard`                | perturbation     | Computes the similarity between predicted and true sets of top differentially expressed (DE) genes.             |
+| `spearman_correlation`   | perturbation     | Rank correlation between predicted and actual expression of masked genes     |
+| `accuracy_score`         | perturbation     | Accuracy of predicting binarized gene expression values     |
+| `f1_score`               | perturbation     | F1-score of predicting binarized gene expression values     |
+| `precision_score`        | perturbation     | Precision of predicting binarized gene expression values     |
+| `recall_score`           | perturbation     | Recall of predicting binarized gene expression values     |
```

Comment:
These metrics are not solely for perturbation, nor are they specifically for gene expression values. They're for any kind of (binary) classification